src/core/stdc/errno.c

```diff
@@ -11,7 +11,11 @@
 
 #include <errno.h>
 
-
+#if __DragonFly__
+int * __error1(void) {
+    return __error();
+}
+#else
```

Comment:
Thinking about this some more, the idea behind the block in `core.stdc.errno` below is to avoid needing this C file from druntime, as implemented in #1917.  Would something like this work to replace the D declaration and alias below, so you could drop this C patch altogether?
```
pragma(mangle, "errno") extern int __errno;
ref int errno() { return __errno;}
```

Reply:
@joakim-noah Thanks for this much nicer solution !

Reply:
Does it work, ie all the stdlib tests pass?

Reply:
Yes it does. I run all the unittests before each/any commit.